DataScience/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Mews data analyst task
+
+## Your task
+
+You have been given a task to look into data of one really nice hotel and find some answers and insights for following questions:
+
+1) What are the popular choices of booking rates (table rate, columns ShortRateName or RateName) for different segments of customers (table reservation, columns AgeGroup, Gender, NationalityCode) 
```

Comment:
I would put table and colum names into backticks (as in Specification paragraph.

Reply:
Ok